src/scenekit.cs

```diff
@@ -1715,10 +1972,19 @@ interface SCNRenderer : SCNSceneRenderer, SCNTechniqueSupport {
 		[iOS (9,0)][Mac (10,11)]
 		[Export ("renderAtTime:viewport:commandBuffer:passDescriptor:")]
 		void Render (double timeInSeconds, CGRect viewport, IMTLCommandBuffer commandBuffer, MTLRenderPassDescriptor renderPassDescriptor);
+
+		[Watch (4,0), TV (11,0), Mac (10,13), iOS (11,0)]
+		[Export ("renderWithViewport:commandBuffer:passDescriptor:")]
+		void RenderWith (CGRect viewport, IMTLCommandBuffer commandBuffer, MTLRenderPassDescriptor renderPassDescriptor);
 #endif
 		[TV (10, 0), Mac (10, 12), iOS (10, 0)]
 		[Export ("updateProbes:atTime:")]
 		void Update (SCNNode [] lightProbes, double time);
+
+		[Watch (4,0), TV (11,0), Mac (10,13), iOS (11,0)]
+		[Export ("updateAtTime:")]
+		void UpdateAtTime (double time);
```

Comment:
Swift named this `update`, and the first argument `atTime`; I think we should do the same.